src/io/flutter/view/MultiIconSimpleColoredComponent.java

```diff
@@ -79,7 +79,7 @@ public PositionedIcon(Icon icon, int index) {
 
   public static final Color SHADOW_COLOR = new JBColor(new Color(250, 250, 250, 140), Gray._0.withAlpha(50));
   @SuppressWarnings("unused") public static final Color STYLE_SEARCH_MATCH_BACKGROUND = SHADOW_COLOR; //api compatibility
```

Comment:
fix some bugs I'd introduced to this class in the logic computing the offset for fragments.